ground/gcs/src/plugins/config/stabilization.ui

```diff
@@ -498,7 +498,7 @@
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
-      <number>1</number>
+      <number>0</number>

```

Comment:
I think this element describes which tab (Basic, Advanced, MultiWii Rate, Expert) is the default one.

I changed that unintentionally to Advanced (1) when i did the expo changes; because when you save the ui file it sets automatically the currently opened tab as default.
Peabody gave me that hint.

So  this is now changed back to basic (0).